lib/compiler/graph-builder.js

```diff
@@ -304,6 +307,10 @@ var GraphBuilder = Base.extend({
         }
 
         return null;
+    },
+    _semantic_ast: function(ast) {
+        var semantic = new SemanticPass({ now: this.now });
```

Comment:
Creating the instance here and repeatedly for each value seems a bit odd, but it is more 'self contained'. What do you think?